src/features/form/__tests__/sagas.test.js

```diff
@@ -610,12 +610,15 @@ describe('form sagas', () => {
   });
 
 
-  // TODO we aren't actually doing this, doesn't work because we prepopulate post with values
-/*  it('should submit delete obs call for any obs not in post', () => {
+  it('should not update encounter datetime if date component does not change', () => {
```

Comment:
Always a great feeling when a feature you are modifying already has good test coverage and it's easy to add a few new cases for your new use case... and you get 90% of the work and testing done before you even after to fire up the app... ;)